_posts/2020-01-07-split-keyboard-one-month.md

```diff
@@ -0,0 +1,26 @@
+---
+layout: post
+title: Split Keyboard - After one month
+date: 2020-01-07 16:28:05.000000000 +01:00
+type: post
+published: true
+status: publish
+categories: [Writing, Programming]
+image:
+image2:
+author: Valentino Urbano
+---
+
+I've now been using the split keyboard for around one month. I do not  use it every day so the actual usage is less, but I try to use it whenever I can.
+
+## Adaptation
+
+For the first two weeks it was really hard to even hit the right letters since I was not use to having the keyboard split in the middle and without any letter printed on the keys so I kept hitting the wrong button. This still happens at times but I'm not always completely error free. The other big problem was that it only has 4 rows of keys and not 5 so it has modifiers to be able to type the remaining special characters, numbers and function keys. This was a whole new keyboard layout to learn. I found that a few keys were not in obvious places so[^1] I re-flashed it and changes a few keys around. You can find the [file with the keymap on my Github][1]. I will also write about the flashing process in a following article.
```

Comment:
use - > used

Reply:
buttonS

Reply:
This still happens at times but I'm not always completely error free.

Um... Both parts of this sentence state the same... The only way it would make sense is if you said that you don't make errors ALL THE TIME